api/GovStack_Registration_BB_API_workflow-0.0.1-resolved.json

```diff
@@ -0,0 +1,525 @@
+					},
+					"code": {
+						"type": "string",
+						"example": "6008"
+					},
+					"message": {
+						"type": "string",
+						"example": "Something is wrong with the data, please check one more time."
+					},
+					"details": {
+						"$ref": "#/components/schemas/ErrorsListDto"
+					}
+				}
+			},
+			"CompletedTaskResponse": {
+				"type": "object",
+				"properties": {
+					"id": {
+						"type": "string",
+						"example": "8a70cd6d-bdb2-11ed-9397-0242ac120004"
+					},
+					"processInstanceId": {
+						"type": "string",
+						"example": "42962de0-bdb2-11ed-9397-0242ac120004"
+					},
+					"rootProcessInstanceId": {
+						"type": "string",
+						"example": "42962de0-bdb2-11ed-9397-0242ac120004"
+					},
+					"rootProcessInstanceEnded": {
+						"type": "boolean",
+						"example": "true"
+					},
+					"variables": {
+						"type": "object",
+						"additionalProperties": {
+							"$ref": "#/components/schemas/VariableValueResponse"
+						}
+					}
+				}
+			},
+			"VariableValueResponse": {
+				"type": "object",
+				"properties": {
+					"type": {
+						"type": "string",
+						"example": "Abc"
+					},
+					"value": {
+						"type": "object",
+						"example": "Test"
+					},
+					"valueInfo": {
+						"type": "object",
+						"additionalProperties": {
+							"type": "object"
+						}
+					}
+				}
+			},
+			"ApplicantTaskResponse": {
+				"type": "object",
+				"properties": {
+					"id": {
+						"type": "string",
+						"example": "8a70cd6d-bdb2-11ed-9397-0242ac120004"
+					},
+					"taskDefinitionKey": {
+						"type": "string",
+						"example": "42962de0-bdb2-11ed-9397-0242ac120004"
+					},
+					"name": {
+						"type": "string",
+						"example": "Application xyz"
+					},
+					"assignee": {
+						"type": "string",
+						"example": "42962de0-bdb2-11ed-9397-0242ac120004"
+					},
+					"status": {
+						"type": "string",
+						"example": "PENDING"
+					},
+
+					"applicantId": {
+						"type": "string",
+						"example": "42962de0-bdb2-11ed-9397-0242ac120004"
+					},
+
+
+
+					"created": {
+						"type": "string",
+						"format": "date-time",
+						"example": "2000-10-23T00:00:00.000Z"
+					},
+					"registered": {
+						"type": "string",
+						"format": "date-time",
+						"example": "2000-10-23T00:00:00.000Z"
+					},
+
+					"description": {
+						"type": "string",
+						"example": "Any text."
+					},
+					"processDefinitionName": {
+						"type": "string",
+						"example": "Post partum registration service"
+					},
+					"processInstanceId": {
+						"type": "string",
+						"example": "42962de0-bdb2-11ed-9397-0242ac120004"
+					},
+					"processDefinitionId": {
+						"type": "string",
+						"example": "42962de0-bdb2-11ed-9397-0242ac120004"
+					},
+					"formKey": {
+						"type": "string",
+						"example": "42962de0-bdb2-11ed-9397-0242ac120004"
+					},
+					"suspended": {
+						"type": "boolean",
+						"example": "true"
+					},
+					"businessKey": {
```

Comment:
what does businessKey refer to?

Reply:
I think it is the same as KEY. we should align it.